ballerina/tests/test.bal

```diff
@@ -0,0 +1,203 @@
+// Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/test;
+import ballerina/log;
+
+configurable string token = ?;
+
+final string artistId = "0TnOYISbd1XYRBk9myaseg";
+final string albumId = "4aawyAB9vmqN3uQ7FjRGTy";
+final string audiobooksId = "7iHfbu1YPACw6oZPAFJtqe";
+final string categoryId = "dinner";
+final string episodeId = "512ojhOuo1ktJprKbVcKyQ";
+final string chaptersId = "0D5wENdkdwbqlrHoaJ9g29";
+final string playlistsId = "3cEYpjA9oz9GiPac4AsH4n";
+final string showsId = "38bS44xjbVVZ3No3ByF1dJ";
+final string tracksId = "11dFghVXANMlKmJXsNCbNl";
+final string userId = "smedjan";
+
+// Initialize the client with auth configuration
+ConnectionConfig config = {
+    auth: {
+        token: token
+    }
+};
+
+final Client spotify = check new(config);
+
+@test:Config {}
+function getAlbumTest() returns error? {
+    
+    AlbumObject|error response = spotify->/albums/[albumId];
+    if response is error {
+        log:printError("Failed to get album", response);
+        test:assertFail("API call failed: " + response.message());
+    }
+    log:printInfo("Test passed: Successfully retrieved album data");
```

Comment:
here we can simply the logic using the `check` expression to avoid the error checking scenarios. Please refer to [Twitter connector tests implementation ](https://github.com/ballerina-platform/module-ballerinax-twitter/blob/04c928d11d137bbae04aba171dcdcb20d1021afc/ballerina/tests/tests.bal#L35) to have a better idea. 

Please update all the test cases accordingly